Scripts/EllosBoonSelectorMod.lua

```diff
@@ -628,15 +766,31 @@ function PredictStartingRoomReward( seedForPrediction, currentSeed )
     -- Get exact boon rewards and rarity and update the menu
     roomReward.BoonData.Options = PredictStartingGodBoonOptions( roomReward.BoonData.God, seedForPrediction, currentSeed )
     roomReward.FirstRoomChaos = PredictChaos(6, seedForPrediction)
+    if roomReward.FirstRoomChaos then
+      roomReward.FirstRoomShrine = false
+    else
+      roomReward.FirstRoomShrine = PredictChaos(7, seedForPrediction)
```

Comment:
We should refactor PredictChaos to be called something else.  Since we are using the same code to check a variety of RNG calls.